components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -76,4 +76,33 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testDefaultBackgroundColor() {
+    // Then
+    XCTAssertEqual(actionSheet.backgroundColor, .white)
+  }
```

Comment:
Ah and this test should also verify that .view.backgroundColor = white by default (along with the other views).

Reply:
Done